slides/developer-guide/working-with-document-properties/get-protection-properties/_index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Get Protection Properties"
+type: docs
+url: /get-protection-properties/
+weight: 50
+---
+## **Introduction**
+The Aspose.Slides allow you to check if the presentation is password protected. The method can be invoked with or without a password parametr. If the password is not provided some properties in the response are unavaliable. 

```

Comment:
The Aspose.Slides (library) **allows** you to check if **a** presentation is password protected. If the password **parameter** is not provided **,** some properties ...